### ISSUE TITLE
Add a CodeChef solution to this repository.

### ISSUE THREAD

**Issue by Vivek-Hotti** [ChefForces-Competitive-Programming/CodechefAnswers#1]
Naming pattern of file : QuestionCode.file-extension . Everybody is welcome to contribute!

**Comment by Samikshahanchate** [comment]
I am a 2 star coder on codechef. I have solved some of the beginner problems there, can I please submit the solutions on this repository.

**Comment by Vivek-Hotti** [comment]
Yes @Samikshahanchate , please go ahead! Just make sure your submitted solutions are not already existing in the repository.

**Comment by Samikshahanchate** [comment]
Yeah sure. Thank you.

**Comment by Samikshahanchate** [comment]
I am not able to make a pull request.

**Comment by Vivek-Hotti** [comment]
I am not sure that is a usual case.
Please refer [this](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) and also choose the option "Compare across Forks" while submitting a PR.
